Loop/Views/SettingsView.swift

```diff
@@ -221,9 +221,11 @@ extension SettingsView {
     }
     
```

Comment:
Sort CGM device list by name. They seemed to be in random order from what I could tell.